src/src/main/java/controller/ProductRegistrationServlet.java

```diff
@@ -0,0 +1,43 @@
+package controller;
+
+import factories.ProductServiceFactory;
+import model.Product;
+import service.ProductService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(value = "/new_product")
+public class ProductRegistrationServlet extends HttpServlet {
+
+    private static final ProductService PRODUCT_SERVICE = ProductServiceFactory.getInstance();
```

Comment:
```suggestion
    private static final ProductService productService = ProductServiceFactory.getInstance();
```
it isn't constant because this field is mutable :)